sql_app/database.py

```diff
@@ -0,0 +1,15 @@
+"""Connection to the SQLite database."""
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./sql_app.db"
+# SQLALCHEMY_DATABASE_URL = "postgresql://user:password@postgresserver/db"
```

Comment:
you can create the db url using environment variables for password and username, pydantic's settings can automatically read env vars.

https://pydantic-docs.helpmanual.io/usage/settings/

Reply:
this is to advanced for me tbh